js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -646,9 +646,9 @@
               advancedTableContent.push(
                 window.modalView.createSelectEntry(
                   'is-satellite-collection',
-                  'Satellite collection',
+                  'SatelliteCollection',
                   '',
-                  'Create satellite collection? This will disable replication factor.',
+                  'Create SatelliteCollection? This will disable replication factor.',
```

Comment:
Is it worth user-friendly?

Looking like this in UI:
![image](https://user-images.githubusercontent.com/3863107/77149267-3e424c80-6aa2-11ea-9cde-e30885a4df99.png)

Reply:
Consistent spelling is important for the Enterprise Edition features, agreed by Jörg